.travis.yml

```diff
@@ -1,17 +1,42 @@
-language: go
+matrix:
+  include:
+    - language: go
+      name: test make process
+      go:
+        - 1.13.x
+      sudo: false
+      cache:
+        directories:
+          - /home/travis/.cache/go-build
+          - /home/travis/gopath/pkg/mod
+      env:
+        - GOPROXY=https://proxy.golang.org/
+      script:
+        - make check
 
-go:
-  - 1.13.x
+    - language: ruby
+      rvm: 2.6
+      name: Site gem vulnerability check
+      sudo: false # route your build to the container-based infrastructure for a faster build
+      cache: bundler
+      before_script:
+        - chmod +x ./hack/site-gem-vulnerability-test/cibuild
```

Comment:
can you just +x the script in this commit?

Reply:
Do you mean to change to the below?
```
script:
  - chmod +x ./hack/site-gem-vulnerability-test/cibuild
  - ./hack/site-gem-vulnerability-test/cibuild
```